### ISSUE TITLE
Create Test Cases for Database interaction (200 pts)

### ISSUE THREAD

**Issue by zholoien** [ucsb-cs56-f18/ucsb-cs56-buy-sell-giveaway#18]
Making test cases for DBController.java in DBControllerTest.java

**Comment by wilsonmui** [comment]
Do you mean something like creating unit tests for your DBController.java?

**Comment by zholoien** [comment]
Yes. Testing the methods in DBController.java : connect(), insert(),delete(), select(), deleteAll(), and selectAll(). Also getting code to compile, and tests to run and fail.

**Comment by wilsonmui** [comment]
200 pts

**Comment by juangonzalez20** [comment]
Wasn't able to commit anything useful for this, but try googling "Testing mongoDB repo using Spring" and you will find some useful things
